### PR TITLE
B3 for Data Reports

### PR DESCRIPTION
**Pull request by biyeun**
migration of data reports to B3

some screenshots
screen shot 2016-02-16 at 9 48 53 pm
screen shot 2016-02-16 at 9 49 02 pm
screen shot 2016-02-16 at 9 49 08 pm
screen shot 2016-02-16 at 9 49 33 pm

### CODE REVIEW

File: corehq/apps/data_interfaces/interfaces.py [hunk 1]
```diff
@@ -57,7 +58,7 @@ def all_case_sharing_groups(self):
     @property
     def headers(self):
         headers = DataTablesHeader(
-            DataTablesColumn(mark_safe('Select  <a href="#" class="select-all btn btn-mini btn-inverse">all</a> <a href="#" class="select-none btn btn-mini btn-warning">none</a>'), sortable=False, span=2),
+            DataTablesColumn(mark_safe('Select  <a href="#" class="select-all btn btn-xs btn-info">all</a> <a href="#" class="select-none btn btn-xs btn-default">none</a>'), sortable=False, span=2),
```
Comment (dimagimon) [inline corehq/apps/data_interfaces/interfaces.py]: E501 line too long (198 > 115 characters)

**Comment by czue** [comment]
wow, cool how little code this is!

:baby_bottle: :blossom: :name_badge:

**Comment by biyeun** [comment]
@czue careful planning and pre-PRs!

**Comment by biyeun** [comment]
QA ticket submitted and feature is on staging

**Comment by biyeun** [comment]
QA is complete cc @benrudolph

**Pull request merged**